src/xcode/ENA/ENA/Source/Scenes/AppClosureNoticeDetails/__tests__/AppClosureNoticeDetailsViewModelTests.swift

```diff
@@ -0,0 +1,52 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+@testable import ENA
+
+class AppClosureNoticeDetailsViewModelTests: CWATestCase {
+
+	func testGIVEN_AppClosureNoticeModel_WHEN_getDynamicTableViewModel_THEN_SectionsAndCellCountsMatch() throws {
+		// GIVEN
+		let cclService = FakeCCLService()
+		
+		let titleText = DCCUIText(
+			type: "string",
+			quantity: nil,
+			quantityParameterIndex: nil,
+			functionName: nil,
+			localizedText: ["de": "Betriebsende"],
+			parameters: []
+		)
+
+		let subtitleText = DCCUIText(
+			type: "string",
+			quantity: nil,
+			quantityParameterIndex: nil,
+			functionName: nil,
+			localizedText: ["de": "Der Betrieb der Corona-Warn-App wird am xx.xx.xxxx eingestellt."],
+			parameters: []
+		)
+		
+		let longText = DCCUIText(
+			type: "string",
+			quantity: nil,
+			quantityParameterIndex: nil,
+			functionName: nil,
+			localizedText: ["de": "Sie erhalten dann keine Warnungen mehr über Risikobegegnungen und können selbst andere nicht mehr warnen. Sie können keine Tests mehr registrieren und erhalten keine Testergebnisse mehr über die App. Auf Ihre Zertifikate und das Kontakt-Tagebuch haben Sie weiterhin Zugriff. Allerdings können Sie keine neuen Zertifikate mehr hinzufügen."],
```

Comment:
I know that's only the test but still I suggest to change "Risikobegegnungen" to "Risiko-Begegnungen".
```suggestion
			localizedText: ["de": "Sie erhalten dann keine Warnungen mehr über Risiko-Begegnungen und können selbst andere nicht mehr warnen. Sie können keine Tests mehr registrieren und erhalten keine Testergebnisse mehr über die App. Auf Ihre Zertifikate und das Kontakt-Tagebuch haben Sie weiterhin Zugriff. Allerdings können Sie keine neuen Zertifikate mehr hinzufügen."],
```